T/TempestRemap/build_tarballs.jl

```diff
@@ -0,0 +1,76 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TempestRemap"
+version = v"2.1.1"
+sources = [
+    ArchiveSource("https://github.com/ClimateGlobalChange/tempestremap/archive/refs/tags/v$(version).tar.gz",
+                  "f5ea21f82b358ba127550fc1f49e701cba0379d22ce030c274135109f678b980"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/tempestremap*
+
+autoreconf -fi
+mkdir -p build && cd build
+
+../configure \
+  --prefix=${prefix} \
+  --build=${MACHTYPE} \
+  --host=${target} \
+  --with-blas=blastrampoline \
+  --with-lapack=blastrampoline
+
+make -j${nproc} all
+make install
+
+install_license ../LICENSE
+"""
+
+# Note: We are restricted to the platforms that NetCDF supports, the library is Unix only
+platforms = [
+    Platform("x86_64", "linux"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("x86_64", "macos"),
```

Comment:
Can we also support `Platform("aarch64", "macos")`?

Reply:
Does NetCDF work on aarch64?

Reply:
Doesn't look like it?  https://github.com/JuliaBinaryWrappers/NetCDF_jll.jl#platforms

Reply:
It's limited by HDF5....

Reply:
there is no end-date to a long time I guess.... https://github.com/JuliaPackaging/Yggdrasil/pull/567